Source/LinqToDB.EntityFrameworkCore/LinqToDBForEFTools.ContextExtensions.cs

```diff
@@ -257,7 +261,51 @@ public static ITable<T> GetTable<T>(this DbContext context)
 
 			return context.CreateLinqToDbContext().GetTable<T>();
 		}
-		
+
+		#endregion
+
+		#region Interceptors
+
+		/// <summary>
+		/// Returns list of registered Linq2Db interceptors from EF Context
+		/// </summary>
+		/// <typeparam name="T">Mapping class type.</typeparam>
+		/// <returns>Db context object</returns>
+		public static IList<IInterceptor> GetLinq2DbInterceptors(this DbContext context)
+		{
+			if (context == null) throw new ArgumentNullException(nameof(context));
+
+			var contextOptions = ((IInfrastructure<IServiceProvider>)context.Database)?
+				.Instance?.GetService(typeof(IDbContextOptions)) as IDbContextOptions;
+
+			return contextOptions?.GetLinq2DbInterceptors() ?? new List<IInterceptor>();

```

Comment:
Return `null` if there is no interceptors.

Reply:
I don't see a point in this, but ok - changed.